src/Tests/UnitTest/ComInteropTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using Windows.ApplicationModel.DataTransfer.DragDrop.Core;
+using Windows.Graphics.Printing;
+using Windows.Media;
+using Windows.Media.PlayTo;
+using Windows.Security.Authentication.Web.Core;
+using Windows.Security.Credentials;
+using Windows.Security.Credentials.UI;
+using Windows.UI.ApplicationSettings;
+using Windows.UI.Input;
+using Windows.UI.Input.Core;
+using Windows.UI.Input.Spatial;
+using Windows.UI.ViewManagement;
+using Xunit;
+
+namespace UnitTest
+{
+    public class ComInteropTests
```

Comment:
maybe include a comment that many of these tests cannot easily be executed without significant test setup, so cases are either compile time only (skipped, to validate types) or expect a runtime exception.